w3af/core/data/context/context/main.py

```diff
@@ -78,16 +82,21 @@ def get_context_iter(data, payload):
 
 class ContextDetectorHTMLParser(HTMLParser):
 
-    def __init__(self, payload):
+    RAW_TEXT_TAG = {
+        'title',
+        'textarea',
+        'plaintext',
+        'xmp',
+        'listing'
+    }
```

Comment:
May be i must leave only `title` and `textarea` because `plaintext`, `xmp` and `listing` are deprecated and can not be interpreted by some browsers.
I'll will research this and make the changes if needed.